[DOCS] Replace `put` with `create or update` in API names (#70330)
elastic/elasticsearch: docs/java-rest/high-level/rollup/put_job.asciidoc

[role="xpack"]
[[java-rest-high-x-pack-rollup-put-job]]
=== Create or update rollup job API

experimental::[]

> The create or update rollup job API can be used to create a new Rollup job
> in the cluster. The API accepts a `PutRollupJobRequest` object

Reviewer: ```suggestion
Creates a new Rollup job or updates an existing one.
The API accepts a `PutRollupJobRequest` object
```

[[java-rest-high-x-pack-rollup-put-rollup-job-execution]]
==== Execution

> The API can be executed through a `RollupClient`

Reviewer: Invoked? You really don't execute APIs. :-)
